checker/tests/nullness/Issue1147.java

```diff
@@ -0,0 +1,16 @@
+// Issue 1147 https://github.com/typetools/checker-framework/issues/1147
+import java.util.StringJoiner;
+
+class Issue1147 {
+
+    public static void main(String[] args) {
+
+        StringJoiner sj = new StringJoiner(",");
+
+        sj.add("a");
+        // Nullness Checker raises false positive warning
```

Comment:
Please remove this comment because it won't be true once your pull request is merged.